build/Version.props

```diff
@@ -22,5 +22,8 @@
 
     <SdkVersion>$(NugetVersion)</SdkVersion>
     <SdkNugetVersion>$(NugetVersion)</SdkNugetVersion>
+
+    <FullNugetVersion Condition=" '$(IsNotOrchestratedPublish)' != 'true' ">$(NugetVersion)</FullNugetVersion>
```

Comment:
This double negative here is kind of hard to read/understand. Could we factor this condition in clearer manner?

Reply:
We need: 'IsNotOrchestratedPublish' for manifest publishing:
https://github.com/dotnet/cli/blob/4cc726eba4de99f247de845b2654388b0be3fb14/build/Publish.targets#L97


So can do this to make it clearer:L
Condition=" '$(IsNotOrchestratedPublish)' == 'false' "